deployments/faucet-env-staging.nix

```diff
@@ -0,0 +1,17 @@
+{ globals, ... }: with (import ./../lib.nix);
```

Comment:
Ben, I also think we should create a new kind of environment -- so far we have:
- `development`, for dev time
- `staging` and `mainnet` for corresponding clusters managed by DevOps

The new environment would be called correspondingly -- `testnet`.

..so effectively, this would add `deployments/faucet-env-testnet.nix`, instead.